src/samples/Sonar.hpp

```diff
@@ -280,6 +284,12 @@ struct Sonar
      */
     void validate();
 
+    /**
+     * Returns the number of bins in each beam
+     * 
+     */
+    uint32_t getBeamBinCount();
```

Comment:
Since we are trying to be very unambiguous ...

```
/** The number of bins in a single beam
 */
uint32_t getSingleBeamBinCount() const;
```

(Note the extra "const")